### ISSUE TITLE
Does version 1.57.0 support jdk 1.8

### ISSUE THREAD

**Issue by jiaorenjie** [grpc/grpc-java#10473]
hi，

when i run java client, it throws exception "java.lang.NoSuchMethodError: java.nio.ByteBuffer.clear()Ljava/nio/ByteBuffer".

i see the sourcecode of ByteBuffer and there is no clear() method in jdk 1.8 version.

So 1.57.0 version does not support jdk 1.8?

Thank you everybody.

**Comment by jiaorenjie** [comment]
hi，
i have seen the release note in 1.57.1. I will try it.

Thank you.

**Comment by ejona86** [comment]
Yeah, sorry about that. I expect 1.57.1 fixes you, so I'm going to go ahead and close. If you have further trouble, comment and I can reopen.
